docs/user-guide/develop_builder_block.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Block
+sidebar_label: Builder/Block
+sidebar_position: 4.3
+---
+
+
+# Block
+
+## General
+
+A Block represents a node within the Directed Acyclic Graph (DAG), each embodying a specific processing logic. Typically, a Block comprises several components:
+- **Block Name**: The name of the block.
+- **Block ID**: The unique identifier of the block.
+- **Copy Block ID**: A feature to copy the block's ID.
+- **Conditional Port**: Accepts True or False outputs from upstream `condition` type blocks, determining whether to execute this block based on the upstream result.
+- **Inport**: The information inputted into this block. Some blocks support multiple inports. Inports can handle `text`, `list`, `dict` data types.
+- **Add an Inport**: Adds an additional inport to the block. Manually added inports support connections of `any` data type.
+- **Outport**: The information outputted from this block. Outports typically support `text`, `list`, `dict`, `boolean` data types.
+- **Configuration**: Specific configuration details for the block, which vary from one block type to another.
```

Comment:
According to https://github.com/pingcap/LinguFlow/pull/50/commits/d090e52b2f324777f6371c367346a932574999a3, we need to rename the Configuration -> Parameter.

Reply:
done